ml/src/main/scala/org/apache/spark/ml/h2o/algos/package.scala

```diff
@@ -0,0 +1,72 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.apache.spark.ml.h2o
+
+import org.apache.spark.h2o.H2OContext
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.SQLContext
+
+package object algos extends Logging{
```

Comment:
Do users always call ```import org.apache.spark.ml.h2o.algo._``` before using an algorithm?

Reply:
Yup, they need to import this package to be able to use our Pipeline stages ( H2OGLM etc)